### PR TITLE
Removes Borg Emag RNG

### PR DESCRIPTION
**Pull request by Fox-McCloud**
Thought about doing this myself, earlier, but looks like Goof beat me to it: https://github.com/tgstation/-tg-station/pull/9893

This is literally the most retarded thing; random 2.5% chance that the borg knows you fail to emag the cover and random 12.5% chance he knows when you fail to emag his interface.

Simply put, emagging a borg's interface isn't lolRNG anymore.

### CODE REVIEW

**Comment by fj45** [comment]
Why why why why!

**Comment by Ultimate-Chimera** [comment]
:-1:

**Comment by tigercat2000** [comment]
Why? 90% chance to succeed the cover unlock, and you'll be told instantly if you failed. And then you can just emag them again with your infinite use emag. This seems like pointless "balancing", which is only going to ever come up rarely.

**Comment by Fox-McCloud** [comment]
@fj45 

Why is this a thing (I don't know) or why remove it? Well, in a word: player agency.

Player agency, that is, giving players control over their actions is a very good thing for gameplay--generally things should have fairly consistent results when an action is undertaken; taking away this agency/control, especially by random chance only introduces frustration. IE; imagine if uploading a law to the AI, patching someone's wounds, or shooting a revolver had a random chance to fail; it's just introducing frustration because a 50% modifier hates you that day (Other example that you may be able to relate to better: unwrenching an atmos pipe fails half the time because "you slip and miss with the wrench!").

Another reason is there's been rising concern amongst administration of the toothlessness of antags; his is a small help to that; traitors will be able to use an emag against a borg without fear of them shouting over radio how they have hostile intrustions in their software---orrr the worse, the borg whacking the person.

@tigercat2000 

Actually, using emags on borgs or their coverlocks never used up an emag charge; but I don't think the number of charges should impact whether something is successful or not (especially when 99% of actions in SS13 generally have consistent results each time you do them--ESPECIALLY with emags).

**Comment by monkeysfist101** [comment]
Nope. Nope. Nope. The tribe has spoken. Nope.

**Comment by Fox-McCloud** [comment]
If you're going to nay-say something, at least provide valid reasoning as @tigercat2000 has done, just "no" doesn't really provide any valid input (positive or negative).

**Comment by monkeysfist101** [comment]
Fine. It dosen't need to be changed. It's fine as it is.

**Comment by Earthdivine** [comment]
I don't see how removing RNG is a bad thing, like fox said saying "nope" or "I don't want it" isn't constructive criticism. The RNG bearly matters to a Borg, your gunna be Flashed or Emagged more then likely, so the RNG giving the finger to the antag while he's trying to subvert you is just plain stupid. 

Also yes, the emag do lose charges to this back when it had a finale amount. Hope you enjoyed your 5 seconds of emag, because RNG said nope.

**Comment by Spacemanspark** [comment]
:-1: Keep it, It made it just a little bit more difficult to create SUPER MASSIVE CYBORG ARMY.

**Comment by TheDZD** [comment]
RNG as artificial "difficulty" is pointless. It doesn't make it any easier to create an army of rogue cyborgs because in all likelihood, if you're going to be emagging a cyborg you're going to be able to stun them or even remove their battery.

tl;dr Pointless RNG sucks, :+1:

**Comment by fj45** [comment]
My why why why comment was on the fact that there was any damned RNG involved.

Nuke pointless RNG.

**Comment by ZomgPonies** [comment]
:+1:

**Comment by Robertha89** [comment]
All you need is a flash, bam, emag once, twice, crowbar, remove battery, emag again, replace battery and close, ofc in general its better to be the roboticist, and have em kill people and git em borged one by one. But I digress. :+1: while RNG is nice in someplaces, the emag has no place for such.
